app/spec/spec_helper.rb

```diff
@@ -0,0 +1,94 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
is this file used anywhere? Or was it autogenerated?

If not used, can you delete it?